CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# v1.24.1 (2024-10-04)
+
+## Release Highlights
+* Update ecs-agent to 1.86.3 [bottlerocket-core-kit#168](https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/168)
```

Comment:
https://github.com/bottlerocket-os/bottlerocket/issues/4186 can we call out it closes this issue?